rubyUtils.gradle

```diff
@@ -254,6 +255,7 @@ task installCustomJRuby(dependsOn: buildCustomJRuby, type: Copy) {
     exclude "**/stdlib/rdoc/**"
     includeEmptyDirs = false
     into "${projectDir}/vendor/jruby"
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
is there any way to set this globally? would it be something we'd like to do?

Reply:
Done - moved to https://github.com/elastic/logstash/blob/0e49b12c2781aff953c7875cf1f79c761ae05b6b/build.gradle#L66